### ISSUE TITLE
[sentry-expo] Sourcemap not found with expo-updates

### ISSUE THREAD

**Issue by RodolfoGS** [expo/expo#10571]
### Summary of Issue 

Seems that this issue https://github.com/expo/sentry-expo/issues/134 is regression with the latest versions of expo, expo-updates and sentry-expo

I'm testing the `3.0.0-rc4`. If you install an update via `expo-updates`, Sentry can't found the sourcemaps.

### Environment - output of `expo diagnostics` & the platform(s) you're targeting

```
  Expo CLI 3.27.13 environment info:
    System:
      OS: macOS 10.15.5
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 14.13.0 - /usr/local/bin/node
      Yarn: 1.19.1 - /usr/local/bin/yarn
      npm: 6.14.8 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.9.3 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: iOS 14.0, DriverKit 19.0, macOS 10.15, tvOS 14.0, watchOS 7.0
      Android SDK:
        Android NDK: 21.3.6528147
    IDEs:
      Android Studio: 4.0 AI-193.6911.18.40.6514223
      Xcode: 12.0/12A7209 - /usr/bin/xcodebuild
    npmPackages:
      expo: ~39.0.2 => 39.0.3
      react: 16.13.1 => 16.13.1
      react-dom: 16.13.1 => 16.13.1
      react-native: ~0.63.3 => 0.63.3
      react-native-web: ~0.13.12 => 0.13.14
    npmGlobalPackages:
      expo-cli: 3.27.13
    Expo Workflow: bare
```

### Reproducible Demo

Repo: https://github.com/RodolfoGS/sentry-test

1. `git clone git@github.com:RodolfoGS/sentry-test.git`
2. `cd sentry-test`
3. `yarn install`
4. Configure the following files with your Sentry values (org, project, token and dsn)
```
 - android/sentry.properties
 - app.json
 - ios/sentry.properties
 - App.js
```
5. `expo publish` (to update `EXPO_UPDATE_URL` property)
6. Build APK as release
7. Install in your device, run and tap on `Crash Now` button
8. The crash appears perfectly on Sentry, with their sourcemaps 👍
9. `expo publish`
10. Tap on `Update Now` button
11. Tap on `Crash Now` button
12. The crash appears on Sentry without their sourcemaps 👎

I'm experimenting this issue on Android and iOS

**Comment by brentvatne** [comment]
@RodolfoGS - did you follow these instructions? https://github.com/expo/sentry-expo#configure-no-publish-builds

**Comment by RodolfoGS** [comment]
Yes I did those steps, I created a demo to reproduce

**Comment by cruzach** [comment]
thanks @RodolfoGS ! I'm looking into this, although I'm not sure what's causing it yet. Can you confirm for me that the `release` and `dist` properties you have are the same on your sourcemaps page:
<img width="274" alt="Screen Shot 2020-10-06 at 5 49 09 PM" src="https://user-images.githubusercontent.com/35579283/95263805-5ee2b180-07fc-11eb-9340-8af932bb0034.png">
and in the individual error's tags?

**Comment by RodolfoGS** [comment]
These screenshots are from the demo that I created.
**Release** tag match with the release created at **expo publish** time.

### Error tags:
![image](https://user-images.githubusercontent.com/870365/95264527-18d61f80-07f5-11eb-85cc-8c94ed3bd481.png)

---

### Artifacts / Sourcemaps:
![image](https://user-images.githubusercontent.com/870365/95264552-255a7800-07f5-11eb-96c7-6f8719e85ec2.png)

**Comment by cruzach** [comment]
Okay, that's exactly what I see as well. This is odd that Sentry isn't showing the error message "sourcemaps not found" which i've definitely seen before 🤔

**Comment by RodolfoGS** [comment]
Yes, I saw that message in the past (**sourcemaps not found**), but now not appear. It's so confused

**Comment by cruzach** [comment]
fix is released in `sentry-expo@3.0.4`
